README.md

```diff
@@ -38,73 +39,132 @@ just install Ketrew:
+Remember that at runtime you'll need `ssh` in your `$PATH` to execute commands on
+foreign hosts.
+
+*Optional*: Ketrew, like any [Lwt](http://ocsigen.org/lwt/)-based piece of
+software, will be much faster and scalable when `libev` is detected and used
+as a backend. Use `opam install conf-libev` to tell opam that `libev` is
 [installed](http://opam.ocaml.org/packages/conf-libev/conf-libev.4-11/), which
-you can ensure with `brew install libev` on MacOSX, `apt-get install libev-dev`
-on Debian/Ubuntu, or `yum install libev-devel` on CentOS (which requires
-`export C_INCLUDE_PATH=/usr/include/libev/` and
-`export LIBRARY_PATH=/usr/lib64/` before `opam install conf-libev`).
+you can ensure with
+
+  - `brew install libev` on MacOSX
+  - `apt-get install libev-dev`on Debian/Ubuntu,
+  - `yum install libev-devel` on CentOS (which requires
+    `export C_INCLUDE_PATH=/usr/include/libev/` and `export LIBRARY_PATH=/usr/lib64/`
+
+before `opam install conf-libev`.
 
 ### Without Opam
 
 See the [development documentation](src/doc/Developer_Documentation.md) to find
-out how to build Ketrew (and its dependencies) from the sources.
+out how to build Ketrew (and its dependencies) from source.
 
 
 Getting Started
 ---------------
 
 Ketrew is very flexible and hence may seem difficult to understand, let's get a
-very minimalistic workflow running.
-
-The first time you use Ketrew, you need to configure it, simplest by calling
-`ketrew init`, and please choose an authentication token:
-
-    rm -fr ~/.ketrew/ # if you had a previous configuration there
-    ketrew init --with-token my-not-so-secret-token
-    
-by default this will configure Ketrew in `$HOME/.ketrew/` with a client/server
-mode **not** using TLS on port `8756` and with a local Sqlite database
-(use the option `--use-database URI` to choose another
-[database backend](src/doc/Database_Backends.md)).
-See `ketrew init --help` for more
-options, you can even ask it to generate self-signed TLS certificates.
-See also the [documentation](src/doc/The_Configuration_File.md)
-on the configuration file learn how to tweak it.
-
-You can check that the client or the server are configured:
-
-    ketrew print-configuration
-    ketrew print-configuration --configuration-profile server
-    ketrew print-configuration -P daemon
-
-You may then start a server:
-
-    KETREW_PROFILE=daemon ketrew start-server
-
-and then open the GUI:
-
-    ketrew gui
-
-which is just trying to load
-<http://127.0.0.1:8756/gui?token=my-not-so-secret-token> ☺
-
-You can always stop the server or check its status:
-
-    ketrew stop -P daemon
-    ketrew status -P daemon
+minimal workflow running.
+
+Before you can use Ketrew, you need to configure it:
+
+    $ ketrew init
+
+By default this will configure Ketrew in
+
+    $ ls $HOME/.ketrew/
+    authorized_tokens	configuration.ml
+
+You can check that the client or the server are configured (the client is returned by default):
```

Comment:
@iskandr That's a fair point. Upon reflection, I'd like to get rid of the earlier mention of a standalone mode. I'd rather users not even know about it. Do you think that the average reader understands the "client"-"server" terms ?